code/modules/mob/living/silicon/silicon_defense.dm

```diff
@@ -72,6 +72,8 @@
 		if ("help")
 			M.visible_message("<span class='notice'>[M] pets [src].</span>", \
 							"<span class='notice'>You pet [src].</span>")
+			if(M.job in list("Research Director", "Scientist", "Roboticist")) //Only true men know taste
+				SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "pet_borg", /datum/mood_event/pet_borg)
```

Comment:
seems to be you should have a COMSIG_ADD_MOOD_EVENT_BORG_PET or something and only have specific jobs listen for it

Reply:
done, thanks for the suggestion.